### PR TITLE
Add cooldowns

### LINKED ISSUE #283: Webinterface - Implement cooldown
The players should be limited in the amount of actions they can perform over a certain amount of time.

This is a subtask of #280 - Webinterface

### PR DESCRIPTION
**Pull request by Geweldig**
This adds the ability to set cooldowns for actions.
This introduces no new PMD/FindBugs/Checkstyle warnings.

Closes #283

### CODE REVIEW

**Comment by ChielBruin** [comment]
Looks good! can be merged

**Comment by WytzeElhorst** [comment]
Ready to merge imo

File: src/main/java/nl/tudelft/contextproject/util/webinterface/WebUtil.java [hunk 1]
```diff
@@ -26,20 +30,13 @@ private WebUtil() {}
...
-				return "spawnenemy";
-			case 4:
-				return "dropbait";
-			default:
-				return "notanaction";
+	public static Action decodeAction(int action) {
+		Action[] actions = Action.values();
+
+		if (action >= 0 && action < actions.length) {
+			return Action.values()[action];
```
Comment (Taeir) [inline src/main/java/nl/tudelft/contextproject/util/webinterface/WebUtil.java]: You could use actions here

**Comment by Taeir** [comment]
Nice use of enums! Looks much cleaner. Can be merged.

**Pull request merged**